### PR TITLE
Add deployment script based on Invoke (Fix #88)

### LINKED ISSUE #88: Reintroducing deployment with Fabric/Invoke
Invoke (by the same author as Fabric) seems to be a library for leveraging task executions, not particularly with system administration and deploying but about all-around task running. ([refer] (http://www.fabfile.org/roadmap.html#invoke-fabric-2-x-and-patchwork).) Thus, I think it would be better to utilize Fabric 2.0 when it's launched instead of using "raw" Invoke.

### PR DESCRIPTION
**Pull request by tkiapril**
Fix #88.
Not yet tested on machine, this PR was created in reference and to get suggestions.
Only the main deploy.py gets its counterpart in Invoke.
Invoke-based deployment scripts are yet to be set to default in Travis, it will be done when the script gains reasonable confidence.

### CODE REVIEW

**Comment by item4** [comment]
[LGTM](http://www.lgtm.in/i/4QddmTYEs)

**Comment by sjw0687** [comment]
Looks promising!

File: setup.py [hunk 1]
```diff
@@ -57,6 +57,8 @@ def readme():
     # Sentry: Log Aggregation
     'raven >= 5.1.1',
     'blinker >= 1.3',
+    # Deployment
+    "invoke == 0.9.0",
```
Comment (dahlia) [inline setup.py]: How about add `invoke` to a separated new list e.g. `deploy_requires`?  If you register these extra requirements to `extras_require` e.g. `extras_require={'tests': tests_require, 'docs': docs_require, 'deploy': deploy_requires}` you would be able to install them by `pip install -e .[deploy]` command.

Comment (tkiapril) [inline setup.py]: @dahlia Roger that.

**Comment by dahlia** [comment]
Looks good to me.  :100:

**Pull request merged**